Remotion/ObjectBinding/Web.ClientScript/Scripts/BocList.ts

```diff
@@ -503,14 +511,26 @@ class BocList
     {
       if (!PageUtility.Instance.IsInDom (scrollableContainer))
         return;
-  
+
+      BocList.FixValidationErrorOverflowSize(scrollableContainer);
       BocList.FixHeaderSize(scrollableContainer);
       BocList.FixHeaderPosition(tableContainer, scrollableContainer);
       setTimeout(resizeHandler, resizeInterval);
     };
     resizeHandler();
   }
 
+  private static FixValidationErrorOverflowSize(scrollableContainer: HTMLElement)
+  {
+    const width = LayoutUtility.GetWidth(scrollableContainer);
+
+    const overflowContainer: HTMLElement | null = scrollableContainer.querySelector('div.bocListTableBlock tr.bocListValidationRow div');
+    if (overflowContainer != null)
```

Comment:
While possibly not a problem, we should not use the type converting operators. So either we do `if (overflowContainer)` or `if (overflowContainer !=== null)`